data/seeds/03-events.js

```diff
@@ -0,0 +1,35 @@
+exports.seed = function(knex, Promise) {
+    return knex('events').del()
+        .then(() => {
+            return knex('events').insert([
+                {
+                    'event_name': 'Chuckee Cheeses',
```

Comment:
I miss those foam obstacle courses.

Reply:
Bahaha, good you can take my kid :P ya'll have a blast lmao